client/reconfiguration/src/poll_based_state_client.cpp

```diff
@@ -36,7 +36,7 @@ concord::messages::ReconfigurationResponse PollBasedStateClient::sendReconfigura
     }
     concord::messages::deserialize(rep.matched_data, rres);
   } catch (std::exception& e) {
-    LOG_ERROR(getLogger(), "error while initiating bft request " << e.what());
```

Comment:
I would say there's a deeper issue here. If an exception occurs it could be from the sent operations or deserialization. If a sent operation throws this could be treated as an error due to internal state inconsistencies. However if deserialization fails because the length of the reply is inconsistent (externally controlled) then this would cause a termination via a failed ConcordAssert inside deserialize. In other words if a byzantine actor sends a message with inconsistent size this will cause a crash failure.

Reply:
We deserialize here the matched_date. this data must be received from at least n-f replicas. 
Hence, there is no way we would have a corrupted reply (unless the quorum itself is corrupted, in this case, I don't think it is very important what we would do)

Reply:
Ok, if that's the case I agree